guides/release/upgrading/index.md

```diff
@@ -1 +1,81 @@
-Coming soon!
+When someone says they are "upgrading" their Ember app, it could mean one of several things, especially if they say that they are "upgrading to Octane."
+This Guide will help fill in the blanks about how to upgrade your app's version, get access to the latest and greatest features, and form a strategy for using Octane's features in your existing apps.
+
+## Routine minor version maintenance upgrades
+
+Let's say you are upgrading an app from `v3.4` to `v3.8`.
+Although you heard there are some new features, your main goal is to keep up with security updates.
+To do this kind of version upgrade, follow the instructions [here in the CLI Guides](https://cli.emberjs.com/release/basic-use/upgrading/).
+That process will make sure that any peer dependencies of the `ember.js` core codebase get upgraded too.
+Automated tools will help you make the right changes to `package.json` and other files.
+The point of a minor version bump is that you shouldn't need to change anything in your app when you upgrade - it should all keep working, whether you choose to adopt new syntaxes and feature, or not.
+
+## Using new features
+
+Once you have upgraded an app's version, some new features may available right of the box.
+Other features may require that you enable them specifically in your app's configuration, since they may change the app's default behavior.
```

Comment:
```suggestion
On the other hand, some features will require that you enable them specifically in your app's configuration, since they may change the app's default behavior.
```